.gitignore

```diff
@@ -1 +1,2 @@
-node_modules/
\ No newline at end of file
+node_modules/
+.env
```

Comment:
- [ ] It's important to don't push the `.env` file. But in this case, as we are learning and we need to be reviewed as students, it's necessary to share with the Pull Request the necessary information to run the project on another machine. Like me now, I'm trying to run the project, but I'm getting errors because there's information missing on the routes while trying to do a `fetch` or `adding` score.
- [ ] Kindly add the necessary logic in your project to make it work on other machines, as we are in the learning process and we need to show others the work done.

![image](https://github.com/Titus-Kiplagat/leaderboard/assets/93630700/54baef05-0900-4948-b3bb-de605c8e6c02)